ddp-workspace/projects/ddp-prion/src/app/app.module.ts

```diff
@@ -76,6 +77,47 @@ config.doLocalRegistration = DDP_ENV.doLocalRegistration;
 config.mapsApiKey = DDP_ENV.mapsApiKey;
 config.auth0Audience = DDP_ENV.auth0Audience;
 config.projectGAToken = DDP_ENV.projectGAToken;
+config.cookies = {
+  cookies: [
+    {
+      type: 'Functional',
+      actions: null,
+      list: [
+        {
+          name: 'auth0',
```

Comment:
Any text that would need to be translated (meaning any user-facing text) should go into the en.json file (the other files are dummy files for now.  For those files, you can either stick the text in Google translate and paste the translations into those files or use the same text as in English but put the two letter country code in front of the string to make it clear that those fields would be translated).

Reply:
Done in baa19cc

Cookies names, descriptions and expiration information are taken from this config and displayed for user 
in the mat-table inside the CookiesPreferencesModalComponent.
I've added final translation of these fields in the mat-cells of that mat-table. 
Also added all new text to es/he/xh.jsons.